meta/editing.md

```diff
@@ -1,5 +1,165 @@
+
+##### Stage 3: Branch
+
+In this stage, you create a new ‘branch’, or temporary version, of the repository on which to make edits. These edits will later be merged with the main repository branch.
+    
+* Navigate in Terminal to the Life Itself Web3 repo which has been cloned to your computer. To do this, type “cd web3” or “cd < directory >” (as in Stage 2). Press enter. (1:49-2:02)
+* Update your local clone of the remote repository. _You don’t need to do this if you’ve only just cloned the repo just now._
+    * Type into Terminal “git pull”. Press enter.
+* Create a branch on which to make edits 
+    * Type “git checkout -b < your_branch_name >”. E.g. you might name your branch “edits”. So you would type “git checkout -b edits”. This command will create a new branch and switch you to this branch. N.B. the branch name cannot contain spaces.
+
+##### Stage 4: Edit
+
+In this stage, you use either a code editor, such as VS Code (see 4a), or Obsidian (see 4b) to view and edit the website’s Markdown files.
+
+###### 4a: VS Code
+
+* Open VS Code
+* Click the ‘Explorer’ icon at the top left hand side of the window. Then click ‘Open Folder’ to open the Life Itself Web3 repo which you cloned. You will then be able to see the list of folders from the repo on the left side of the window. (2:03-3:35)
+* Find the file that corresponds with the page you want to edit.* (3:31-4:03)
+* Make your edits and save
+
+
+###### 4b: Obsidian
+
+* Open Obsidian
+* Open the cloned github repo folder as a vault (10:07)
+* Find the file that corresponds with the page you want to edit.* 
+* Make your edits and save. 
+    * See video tutorial (13:05-15:26) for info on useful Obsidian features such as shortcuts for linking to other pages.
+
+*N.B. To find the file that corresponds with the page you want to edit, it might help to look at the page URL. E.g. To find the file containing the “Blockchain” page ([https://web3.lifeitself.us/concepts/blockchain](https://web3.lifeitself.us/concepts/blockchain)), go to the folder “concepts“, then the file “blockchain.md”.
+
+##### Stage 5: Pull request
+
+In this stage, you prepare the changes you have made (and saved) to a Markdown file to be published on the website.
+
+* Go to your computer’s Terminal (4:59-5:08)
+* Type “git status”. Press enter. (5:09-5:30)
+* Type “git add < name of modified file >”. E.g. “git add site/content/test.md”. Press enter. (5:31-6:06) If you’ve edited more than one file, you can type “git add .” to add all the modified files with one command.
+* Type “git commit -m “< description of edit >””. E.g. “git commit -m “fix typo”” or “git commit -m “add extra text to definition””.  Press enter. Note that the description of the edit must be in double quotation marks. (6:07-7:01)
+* Type “git push origin <your_branch_name>”, replacing <your_branch_name> with the name of the branch you create.
```

Comment:
@EilidhRoss1 use back ticks instead of quotes when using `< >` or else it will try to parse it as an html element. For eg. replace double quotes here `“git push origin <your_branch_name>”` with back ticks

```md
`git push origin <your_branch_name>`
```

Reply:
thanks Khalil, I've tried again, will keep an eye on it and see if it works